src/libs/actions/IOU.ts

```diff
@@ -6835,8 +6836,18 @@ function canIOUBePaid(
     const isAutoReimbursable = policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES ? false : ReportUtils.canBeAutoReimbursed(iouReport, policy);
     const shouldBeApproved = canApproveIOU(iouReport, chatReport, policy);
 
+    const isFutureTravelExpenseReport = ReportUtils.isPayAtEndExpenseReport(iouReport?.reportID, transactions);
```

Comment:
```suggestion
    const isPayAtEndExpenseReport = ReportUtils.isPayAtEndExpenseReport(iouReport?.reportID, transactions);
```

Any reason to name this differently than the function?

Reply:
Missed that when refactoring,thanks!